### ISSUE TITLE
Game crash when clicking on "Choose Map": Exception of type `System.IO.InvalidDataException`: Unknown binary map format

### ISSUE THREAD

**Issue by blarg227** [OpenRA/OpenRA#7494]
Whenever I attempt to select a different map, the game crashes. For whatever reason, it seems to only happen on my Desktop pc at home.
```
Tiberian Dawn Mod at Version release-20141029
Operating System: Windows (Microsoft Windows NT 6.2.9200.0)

Exception of type `System.IO.InvalidDataException`: Unknown binary map format
   at OpenRA.Map.LoadMapTiles()
   at OpenRA.Map.<Map>c__AnonStorey2.<>m__2()
   at System.Lazy`1.CreateValue()
   at System.Lazy`1.LazyInitValue()
   at System.Lazy`1.get_Value()
   at OpenRA.Graphics.Minimap.TerrainBitmap(TileSet tileset, Map map, Boolean actualSize)
   at OpenRA.MapCache.LoadAsyncInternal()
   at System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()
```

**Comment by LipkeGu** [comment]
The Problem is: The Maps what you have tried to load was updated to a newer format! Thats why your Game is crashing!
 
The Map format has been changed & updated
I prefer that you update your client :)

**Comment by ihptru** [comment]
Or clear all downloaded maps.
They are possibly here:
```
Windows: My Documents\OpenRA\maps
Mac OSX: ~/Library/Application Support/OpenRA/maps
Linux: ~/.openra/maps
```
If it does not help, try also reinstall the game.

**Comment by blarg227** [comment]
Thank you guys! It works now!

**Comment by ihptru** [comment]
Engine should ignore unsupported maps.

**Comment by pchote** [comment]
"fixed" by release-20150424.  We really should put some effort into improving our robustness against bogus maps, though.
